setup.cfg

```diff
@@ -40,6 +40,7 @@ install_requires =
 	janus
 	ansible-runner
 	websockets
+        drools_jpy
```

Comment:
This line is not needed because you have defined the dependency in the setup.cfg. isn't?

Reply:
@Alex-Izquierdo This is in setup.cfg

Reply:
@mkanoor  Sorry, I mistake the line. I meant in the dockerfile: https://github.com/benthomasson/ansible-events/pull/144/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R19

Reply:
👍